src/firefly/js/ui/ImageSelect.jsx

```diff
@@ -351,11 +353,37 @@ const toFilterSummary = (master, key, desc) => Object.entries(countBy(master, (d
 /*--------------------------- Data Product List ---------------------------------------*/
 
 function DataProductList({filteredImageData, groupKey, multiSelect}) {
+    // Create a field for storing default images in rgb field group
+    const {fireValueChange: fireDefaultImagesChange} = useFieldGroupConnector({fieldKey: 'defaultImages', groupKey: FG_KEYS.rgb});
+    const [getDefaultImages, ] = useFieldGroupValue('defaultImages', FG_KEYS.rgb);
```

Comment:
This is a great idea but `useFieldGroupConnecter()` is really specialize and is use for creating field group components. I think you could simplify it with just `useFieldGroupValue()`
```js
const [getDefaultImages, setDefaultImages] = useFieldGroupValue('defaultImages', FG_KEYS.rgb);
```

Reply:
I used useFieldGroupConnector because the FG_KEYS.rgb field group doesn't exist already in store unlike other field groups that get created by FieldGroup component. Didn't realise setter in useFieldGroupValue can also create a field group if it doesn't exist. 

Updated it now, thanks!